src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputOperatorWrap.java

```diff
@@ -61,13 +61,13 @@ void testGenerics1() {
         9) { /* ignore */ }
 
     while (init == // violation ''==' should be on a new line.'
-        10) { }
+        10) { } // violation 'Empty blocks should have no spaces.'
```

Comment:
can we add extended message here too ?
and in all other places, to make it clear that `{}` is expected.

Reply:
done.